bigframes/session/__init__.py

```diff
@@ -749,6 +750,26 @@ def read_gbq_table(
             filters=filters,
         )
 
+    def read_gbq_table_streaming(
+        self, table: str
+    ) -> bigframes.streaming.StreamingDataFrame:
+        """Turn a BigQuery table into a StreamingDataFrame.
+
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> sdf = bpd.read_gbq_table_streaming("bigquery-public-data.ml_datasets.penguins")
+        """
+        from bigframes import streaming
```

Comment:
Please make sure we raise a PreviewWarning and mark this as a preview method in the docstring.

Reply:
Done